tests/integration_tests/tests/agent_tests/test_list_agents.py

```diff
@@ -0,0 +1,101 @@
+########
+# Copyright (c) 2013-2019 Cloudify Platform Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+from integration_tests import AgentTestCase
+from integration_tests.tests.utils import (get_resource as resource,
+                                           create_rest_client)
+
+
+def _is_community():
```

Comment:
Maybe you can use instead of this function: `from manager_rest import premium_enabled`